dlt-database/entity/0001-init_db/User.ts

```diff
@@ -0,0 +1,39 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, OneToMany, JoinColumn } from 'typeorm'
+
+import { Account } from './Account'
+
+@Entity('users', { engine: 'InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci' })
+export class User extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({
+    name: 'gradido_id',
+    length: 36,
+    nullable: true,
+    unique: true,
+    collation: 'utf8mb4_unicode_ci',
+  })
+  gradidoID?: string
+
+  @Column({ name: 'derive1_pubkey', type: 'binary', length: 32, unique: true })
+  derive1Pubkey: Buffer
+
+  @Column({
+    name: 'created_at',
+    type: 'datetime',
+    default: () => 'CURRENT_TIMESTAMP',
+  })
+  createdAt: Date
+
+  @Column({
+    name: 'confirmed_at',
+    type: 'datetime',
+    nullable: true,
+  })
+  confirmedAt?: Date
+
+  @OneToMany(() => Account, (account) => account.user)
+  @JoinColumn({ name: 'user_id' })
+  accounts?: Account[]
```

Comment:
As i remarked above in the Accounts.ts file

i would distingue between the main age-account per 1:1 relation and all others with this existing accounts-relation. Because then there is nothing to search or filter in object-model during runtime, because the database gaves you the main age-account directly...

feel free to get your experiences with it. its only my suggestion...